### PR TITLE
Translations generated in cockpit-machines repo with make update-po

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: machines.html.pot [hunk 1]
```diff
@@ -0,0 +1,11 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE_VERSION\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Cockpit html2po\n"
+
+#: src/index.html:4
+msgid "Virtual machines management"
```
Comment (marusak) [inline machines.html.pot]: Please don't add those partial .pot files

**Review by marusak — DISMISSED** [review]
Please no partial.pot files and also I think this was generated from cockpit repo but it should be generated from https://github.com/cockpit-project/cockpit-weblate to include also languages that are not translated to 50% and thus not in cockpit repo directly.

**Comment by KKoukiou** [comment]
> Please no partial.pot files and also I think this was generated from cockpit repo but it should be generated from https://github.com/cockpit-project/cockpit-weblate to include also languages that are not translated to 50% and thus not in cockpit repo directly.

you got it right, did not realize there was this difference!

File: machines.html.pot (hunk 1 above)
Comment (KKoukiou) [reply machines.html.pot]: Done.

**Comment by marusak** [comment]
Looks good, the only issue is that all unused translations are there as well, just commented out. They take 70% of files unnecessary. Can you please remove those? (I guess some mdg* tool will do that with correct option or just you can find the last translated string and delete till the end of file).

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**